src/Compilers/VisualBasic/Test/Symbol/SymbolsTests/Metadata/PE/LoadingAttributes.vb

```diff
@@ -18,6 +18,39 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.UnitTests.Symbols.Metadata.PE
         Inherits BasicTestBase
 
         <Fact>
+        Public Sub TestDecodeCustomAttributeType()
+            Dim text = <compilation>
+                           <file name="a.vb">
+                               <![CDATA[
+Imports System.Runtime.InteropServices
+
+<CoClass(GetType(Integer))>
+Public Interface IT
+    Sub M()
+End Interface
+]]>
+                           </file>
+                       </compilation>
+            Dim comp1 = CreateCompilationWithReferences(text, references:={MscorlibRef_v20})
+            Dim ref1 = comp1.EmitToImageReference()
+            Dim comp2 = CreateCompilationWithReferences(
+                <compilation>
+                    <file name="a.vb">
+                        <![CDATA[
+Public Class C
+    Implements IT
+    Sub M() Implements IT.M
+    End Sub
+End Class]]>
+                    </file>
+                </compilation>, references:={MscorlibRef_v4_0_30316_17626, ref1})
+            CompileAndVerify(comp2, symbolValidator:=Sub(m As ModuleSymbol)
```

Comment:
In addition, consider also testing with compilation that takes a comp1 as a compilation reference rather than an image reference. This will test Retargeting symbols.